conn.go

```diff
@@ -165,3 +168,8 @@ func Validate(request Message, replies []Message) error {
 
 	return nil
 }
+
+// Config contains options for the netlink socket
+type Config struct {
+	NLGroups uint32 // netlink multicast groups
```

Comment:
How about just `Groups`?  I have been dropping `NL` prefixes since this package is referred to as `netlink` externally.

Also, let's move the comment above the field and have it say something like

```
// Groups is a bitmask which specifies multicast groups.  If set to 0,
// no multicast group subscriptions will be made.
```